app/models/bulkrax/csv_entry.rb

```diff
@@ -76,25 +76,30 @@ def add_file
     end
 
     def build_export_metadata
+      hyrax_record
```

Comment:
@sephirothkod Do we need to call `#hyrax_record` here? If so, why?

Reply:
no we don't. I'll remove it.